core/src/main/kotlin/org/evomaster/core/problem/graphql/builder/GraphQLActionBuilder.kt

```diff
@@ -0,0 +1,1011 @@
+                    isKindOfFieldNameOptional = element.isKindOfFieldNameOptional,
+                    enumValues = element.enumValues,
+                    unionTypes = element.unionTypes,
+                    interfaceTypes = element.interfaceTypes,
+                    isFieldNameWithArgs = element.isFieldNameWithArgs,
+                )
+
+                val template = getInputGene(state, history, maxNumberOfGenes, copy)
+                fields.add(template)
+            } else
+                if (element.KindOfFieldName.lowercase() == GqlConst.LIST) {
+                    val copy = copyTableElement(element, element)
+                    val template = getInputGene(state, history, maxNumberOfGenes, copy)
+
+                    fields.add(template)
+                } else
+                    if (element.kindOfFieldType.lowercase() == GqlConst.INPUT_OBJECT) {
+                        val copy = copyTableElement(element, element)
+                        val template = getInputGene(state, history, maxNumberOfGenes, copy)
+
+                        fields.add(template)
+
+                    } else if (element.kindOfFieldType.lowercase() == GqlConst.ENUM) {
+                        val copy = element.copy(
+                            fieldType = element.typeName,
+                            KindOfFieldName = element.kindOfFieldType,
+                            kindOfFieldType = element.kindOfFieldType,
+                            typeName = element.fieldName,
+                            isKindOfFieldTypeOptional = element.isKindOfFieldTypeOptional,
+                            isKindOfFieldNameOptional = element.isKindOfFieldNameOptional,
+                            enumValues = element.enumValues,
+                            unionTypes = element.unionTypes,
+                            interfaceTypes = element.interfaceTypes,
+                            isFieldNameWithArgs = element.isFieldNameWithArgs
+                        )
+                        val template = getInputGene(state, history, maxNumberOfGenes, copy)
+
+                        fields.add(template)
+                    }
+        }
+        return ObjectGene(element.fieldName, fields)
+    }
+
+    private fun copyTableElement(
+        element: Table,
+        secondElement: Table
+    ): Table {
+        return element.copy(
+            fieldType = element.fieldType,
+            KindOfFieldName = element.KindOfFieldName,
+            kindOfFieldType = element.kindOfFieldType,
+            typeName = element.fieldType,
+            isKindOfFieldTypeOptional = secondElement.isKindOfFieldTypeOptional,
+            isKindOfFieldNameOptional = secondElement.isKindOfFieldNameOptional,
+            enumValues = element.enumValues,
+            unionTypes = element.unionTypes,
+            interfaceTypes = element.interfaceTypes,
+            isFieldNameWithArgs = element.isFieldNameWithArgs
+        )
```

Comment:
see previous comment. check all of theses cases in this file. ie, if copied value is not changed, no need to manually assign it, isn't it?

Reply:
Done